proof-systems/src/darlin/tests/mod.rs

```diff
@@ -418,6 +551,12 @@ mod test {
             committer_key_g2, verifier_key_g2
         ) = get_keys::<_, _, Blake2s>(&params_g1, &params_g2);
 
+        //Generate fake params
+        let mut params_g1_fake = TestIPAPCDee::setup_from_seed(segment_size - 1, b"FAKE PROTOCOL").unwrap();
+        params_g1_fake.copy_params(&params_g1);
```

Comment:
I can't find this function in poly-commit/negative_tests. Have you pushed it ?
Also you didn't update the dependencies in the Cargo.toml file to point to marlin and poly commit branches "negative_tests"

Reply:
I've synced the branches, now function is present in poly-commit